.taskcluster.yml

```diff
@@ -73,6 +73,10 @@ tasks:
         - name: taskcluster-hooks
         - name: taskcluster-index
         - name: taskcluster-notify
+          image: taskcluster/node-and-postgres:node${node}-pg11
+          command: >-
+            ./db/test-setup.sh &&
+            yarn workspace taskcluster-notify coverage:report
```

Comment:
Why not just make all services do this db setup stuff now so that we don't have to expand out the `command` for this. They'll all need to do it soon anyway.